src/pt/pinkseaunicorn/src/eu/kanade/tachiyomi/extension/pt/pinkseaunicorn/PinkSeaUnicorn.kt

```diff
@@ -16,7 +19,19 @@ class PinkSeaUnicorn : Madara(
 
     override val client: OkHttpClient = super.client.newBuilder()
         .rateLimit(1, 2, TimeUnit.SECONDS)
+        .addInterceptor(::checkPasswordProtectedIntercept)
```

Comment:
you can `addNetworkInterceptor` which will receive the redirect url in chain.request, so you can just test that.

Reply:
> you can `addNetworkInterceptor` which will receive the redirect url in chain.request, so you can just test that.

You mean basically changing the `.addInterceptor(::checkPasswordProtectedIntercept)` to `.addNetworkInterceptor(::checkPasswordProtectedIntercept)`?

Reply:
yes and check chain.request().url instead of first getting the response